.github/workflows/checks.yaml

```diff
@@ -23,3 +23,12 @@ jobs:
       - name: goimports-check
         run: |
           make goimports-check
+      - name: Run Tests
```

Comment:
Can you please add a target "test" in the Makefile and use it here? Users can run `make test` to ensure their changes are not breaking anything.

Reply:
Thanks for the review , could you please look at the latest changes and suggest anymore changes if needed

Reply:
sure, I will take a look.

Reply:
Thanks @ARYPROGRAMMER and @Avish34 . `make test` can also be replaced here: https://github.com/CRED-CLUB/propeller/blob/main/.github/workflows/build.yaml#L21

Reply:
Thanks @Abhishekvrshny , checked on that one